src/components/_global/BalTable/BalTable.vue

```diff
@@ -14,44 +14,50 @@
             v-for="(column, columnIndex) in filteredColumns"
             :key="`header-${column.id}`"
             :class="[
-              'p-6 bg-white headingShadow border-b border-gray-200',
+              'p-6 bg-white headingShadow border-b border-gray-100',
```

Comment:
There is a default border color, so you don't always have to set it.
```suggestion
              'p-6 bg-white headingShadow border-b',
```